src/store/actions/index.js

```diff
@@ -0,0 +1,77 @@
+import { axiosWithAuth } from "../../utils/axiosWithAuth";
+
+const urlServer = "http://localhost:5000/api";
+
+export const LOGIN_START = "LOGIN_START";
```

Comment:
This action file is really clean. I Like the exports sorted in with each action creator like that. Looks good. 🚀